optaplanner-docs/src/main/asciidoc/ConstraintStreams/ConstraintStreams-chapter.adoc

```diff
@@ -338,6 +338,13 @@ the prescribed ordering.
 For a full list of all supported `Joiner` implementations and their various overloads, refer to the
 `org.optaplanner.core.api.score.stream.Joiners` class.
 
+[NOTE]
+====
+In case you only need to act when a particular fact exists, and you don't need to access the fact itself, use
```

Comment:
"a particlar fact": is that the first or the second fact?
"you don't need to" is indirection
This sentance can be more concise I think.
Also we need to mention the "one or more" difference.
How about something like

If the otherStream potentially matches multiple times, but it must only be propagated once, use <<constraintStreamsConditionalPropagation,ifExists>> instead.